client/homebrew/navbar/account.navitem.jsx

```diff
@@ -28,7 +29,29 @@ const Account = createClass({
 		};
 	},
 
+	localLogin : async function(){
+		const username = prompt('Enter username:');
+		if(!username) {return;};
+
+		const expiry = new Date;
+		expiry.setFullYear(expiry.getFullYear() + 1);
+
+		const token = await request.post('/login')
+				.send({ username })
+				.then((response)=>{
+					return response.body;
+				})
+				.catch((err)=>{
+					return null;
```

Comment:
```suggestion
					console.warn(err);
```

Reply:
Committed both suggested changes, and resolved the new merge conflict.